vortex-duckdb/src/exporter/cache.rs

```diff
@@ -16,6 +16,7 @@ use crate::duckdb::Vector;
 #[derive(Default)]
 pub struct ConversionCache {
     pub values_cache: DashMap<usize, (ArrayRef, Arc<Mutex<Vector>>)>,
+    pub canonical_cache: DashMap<usize, (ArrayRef, Canonical)>,
```

Comment:
Why do we actually store the ArrayRef in the values pair, besides Canonical?

Reply:
Is it to ensure no other array gets allocated for that address? If so let's add that to the docs.